### ISSUE TITLE
Error parsing a stitching vector

### ISSUE THREAD

**Issue by agerardin** [PolusAI/filepattern#31]
Filepattern throws an error while trying to infer a pattern for a what seems a valid stitching vector :

```python
pattern = fp.infer_pattern(STITCHING_VECTOR_PATH)
```

throws 

```
'PosixPath' object is not iterable
```

Below is a link to the stitching vector

[stitching vector](https://jh.scb-ncats.io/hub/user-redirect/lab/tree/shared/data/IDR/visualization_outputs/Idr0130/montage/190808-3-38-HSV-3C_Plate_14026/img-global-positions-0.txt)

Parsing similar vector worked just fine, for example [this one](https://github.com/usnistgov/MIST/wiki/testdata/Small_Phase_Test_Dataset_Example_Results.zip)

**Comment by agerardin** [comment]
Actually not a bug, but it is rather a distribution issue related to #10.
On osx, Filepattern needs to be build manually as there is no packaged version available for version 2.
However by default, pip will reinstall 1.4.7 in place of the local build if tight requirements are not in place 
(ex : filepattern=^2.0.0)
Thus the error message above popping up as filepattern 1.4.7 had replace the local 2.0.1 build when installing a dependency through poetry (poetry check that all dependencies are met before adding a new package and try to fix the environment).

I would guess one way to address this would be to find a way to add prebuilt binaries for apple silicone.

**Comment by sameeul** [comment]
Fixed in v2.0.3
